src/renderer/entities/notification/ui/NotificationRow/NotificationBody.tsx

```diff
@@ -0,0 +1,119 @@
+import { Trans } from 'react-i18next';
+import { useUnit } from 'effector-react';
+
+import {
+  MultisigAccountInvitedNotification,
+  MultisigNotification,
+  MultisigNotificationType,
+  Notification,
+  ProxyCreatedNotification,
+  ProxyNotificationType,
+} from '@entities/notification';
+import { WalletType } from '@shared/core';
+import { WalletIcon, walletModel } from '@entities/wallet';
```

Comment:
⚠️ wallet entity cross import
⚠️ chain entity import down below

This must raise a question: "Do all these notification UI components belong to an entity or it's a page components?" It's much easier to start from page and decompose everything to widgets/features/entities 😉

Reply:
Changed `NotificationBody` to `NotificationProvider` and moved it to feature with components for each supported notification. Moved `NotificationRow` to page components